### PR TITLE
Implement certificates endpoint filtering by type and my team

### PR DESCRIPTION
**Pull request by gonzalpi**
Endpoint `/api/v1/certificates/:type` returns list of certificates filtered by `type`: `all`, `ibm`, `industry`, `my_teams`

This pull request also includes documentation

### CODE REVIEW

**Comment by emiliogonmar2002** [comment]
> 

Does the endpoint receives a JSON? or is it a dynamic route?

File: README.md [hunk 1]
```diff
@@ -183,6 +183,14 @@ Note: if an endpoint is not listed here, a complete list can be retrieved by run
 
 
 - `GET /api/v1/certificates`: Fetch all certificates
```
Comment (Alyxhelpme) [inline README.md:185]: I think it should be important to specify the response the url gives when not adding the optional Json, front-end team needs all these explanations with pebbles.

**Comment by gonzalpi** [comment]
@emiliogonmar2002 it's dynamic, it can receive
- no JSON
- "type" only
- "my_team" only
- "type" and "my_team"

File: README.md (hunk 1 above)
Comment (gonzalpi) [reply README.md:185]: done, i've now documented that in Certificates section

**Review by Alyxhelpme — APPROVED** [review]
Very gut

**Pull request merged**